test/ToStringTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
Will also need a test to cover added `__invoke` method and data provider cases covering `Stringable` objects